GameApplication/GameApplication/Startup.cs

```diff
@@ -35,6 +35,11 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
+            // Uncomment while using Linux <3
+            // MS SqlServer throws NotSupported on Linux platform, so I temporarily changed it to Sqlite
```

Comment:
<3

Reply:
Yeah, it could be useful when migrating to Docker <3